recipe/meta.yaml

```diff
@@ -13,25 +13,26 @@ package:
 source:
   # only pull source after PyPI release.
   url: https://pypi.org/packages/source/j/jax/jax-{{ version }}.tar.gz
-  sha256: c0c986993026b10bf6f607fecb7417377460254640766ce40f1fef3fd139c12e
+  sha256: 43bae65881628319e0a2148e8f81a202fbc2b8d048e35c7cb1df2416672fa4a8
 
 build:
-  number: 1
+  number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python {{ python_min }}
     - setuptools
     - pip
   run:
+    - opt_einsum
     - python >={{ python_min }}
     - numpy >=1.24
     - opt-einsum
     - scipy >=1.10
     - ml_dtypes >=0.4.0
-    - jaxlib >=0.4.34,<=0.4.35
+    - jaxlib >=0.4.38,<=0.4.38
     - importlib-metadata >=4.6
   run_constrained:
     - cudnn >=9.2.1.18
```

Comment:
![image](https://github.com/user-attachments/assets/c75286ee-dd55-47be-8f2c-7e4ad186e8b4)

https://github.com/jax-ml/jax/blob/jax-v0.4.38/setup.py#L57-L65

Reply:
I'd be inclined to bump both `numpy` and `scipy` to the minimum required by Python 3.12